closed/get_j9_source.sh

```diff
@@ -57,10 +57,16 @@ declare -A shas
 declare -A references
 
 git_urls[openj9]=https://github.com/eclipse-openj9/openj9
-branches[openj9]=master
-
 git_urls[omr]=https://github.com/eclipse-openj9/openj9-omr
-branches[omr]=openj9
+
+currentbranch=`git rev-parse --abbrev-ref HEAD`
```

Comment:
Suggest using `$()` instead of \`\` and that `git branch --show-current` is a clearer way to query the name of the active branch:
```
currentbranch=$(git branch --show-current)
```

Reply:
Should I change OpenJ9.gmk to use --show-current in `OPENJ9_BRANCH := $(shell $(GIT) -C $(OPENJ9_TOPDIR) rev-parse --abbrev-ref HEAD)`?

Reply:
According to our [build instructions for macOS](https://github.com/eclipse-openj9/openj9/blob/master/doc/build-instructions/Build_Instructions_V8.md#1-prepare-your-system-3), git is expected to be version 2.19.2 (or better),but unfortunately, `git branch --show-current` was only added in 2.22, so perhaps we need to stick with `git rev-parse --abbrev-ref HEAD` (here and in `OpenJ9.gmk`).

Reply:
Updated.